### PR TITLE
TELCODOCS-324: IPI Expanding Cluster Without Provisioner Node

### PR DESCRIPTION
**Pull request by kdrake20**
Fixes: https://issues.redhat.com/browse/TELCODOCS-324

For: Version 4.8+

Doc Preview: https://deploy-preview-41734--osdocs.netlify.app/openshift-enterprise/latest/installing/installing_bare_metal_ipi/ipi-install-expanding-the-cluster.html#preparing-the-bare-metal-node_ipi-install-expanding

Signed-off-by: Katie Tothill ktothill@redhat.com

### CODE REVIEW

File: modules/ipi-install-preparing-the-bare-metal-node.adoc [hunk 1]
```diff
@@ -17,44 +17,30 @@ Some administrators prefer to use static IP addresses so that each node's IP add
...
-[source,bash]
-----
-[kni@provisioner ~]$ sudo cp oc /usr/local/bin
-----
+. Power off the bare metal node by using the baseboard management controller, and ensure it is off.
 
-. Power off the bare metal node via the baseboard management controller and ensure it is off.
-
-. Retrieve the user name and password of the bare metal node's baseboard management controller. Then, create `base64` strings from the user name and password. In the following example, the user name is `root` and the password is `calvin`.
+. Retrieve the user name and password of the bare metal node's baseboard management controller. Then, create `base64` strings from the user name and password:
```
Comment (lpettyjo) [inline modules/ipi-install-preparing-the-bare-metal-node.adoc:24]: You should either make the second sentence a second step, or recast as: "...the bare metal node's baseboard management controller, and then, create `base64` strings from the user name and password:

File: modules/ipi-install-provisioning-the-bare-metal-node.adoc [hunk 2]
```diff
@@ -60,44 +64,50 @@ openshift-worker-1.example.com      1         1         1       1           55m
...
 +
-[source,bash]
+[source,terminal,subs="+quotes"]
 ----
 $ oc -n openshift-machine-api get bmh openshift-worker-<num>
 ----
 +
-Where `<num>` is the worker node number. The status changes from `ready` to `provisioning`.
+where:
+`<num>`:: Specifies the worker node number, whose status changes from `ready` to `provisioning`.
```
Comment (lpettyjo) [inline modules/ipi-install-provisioning-the-bare-metal-node.adoc:84]: Remove the comma.

**Review by lpettyjo — COMMENTED** [review]
Please provide in the first comment box for the PR:
- OCP versions the PR should apply against.
- Description of PR.
- Link to bug or JIRA card.
- Link to the areas preview areas that are changed by this PR.
Thanks!

File: modules/ipi-install-preparing-the-bare-metal-node.adoc (hunk 1 above)
Comment (ktothill) [reply modules/ipi-install-preparing-the-bare-metal-node.adoc:24]: Updated as an additional step as suggested.

File: modules/ipi-install-provisioning-the-bare-metal-node.adoc (hunk 2 above)
Comment (ktothill) [reply modules/ipi-install-provisioning-the-bare-metal-node.adoc:84]: Updated as suggested.

File: modules/ipi-install-provisioning-the-bare-metal-node.adoc [hunk 3]
```diff
@@ -60,44 +64,50 @@ openshift-worker-1.example.com      1         1         1       1           55m
...
 +
-[source,bash]
+[source,terminal,subs="+quotes"]
 ----
 $ oc scale --replicas=<num> machineset <machineset> -n openshift-machine-api
 ----
 +
-Replace `<num>` with the new number of worker nodes. Replace `<machineset>` with the name of the machine set from the previous step.
+where:
+`<num>`:: Specifies the new number of worker nodes.
```
Comment (ogradyp) [inline modules/ipi-install-provisioning-the-bare-metal-node.adoc:73]: Should this not read 'Specifies the number of new worker nodes.' 
I suppose it depends on whether the nodes themselves are new or if there is a pool of existing nodes and the number of nodes is adjusted (new), as needed.

File: modules/ipi-install-removing-the-provisioner-node.adoc [hunk 4]
```diff
@@ -0,0 +1,13 @@
+// This is included in the following assemblies:
+//
+// installing/installing_bare_metal_ipi/ipi-install-post-installation-configuration.adoc
+
+[id="ipi-install-removing-the-provisioner-node_{context}"]
+= Removing the provisioner node
+
+Optionally, the provisioner node can be removed after installation.
+Make sure to back up the following files:
```
Comment (ogradyp) [inline modules/ipi-install-removing-the-provisioner-node.adoc:9]: The three listed items are not all files - 1 file, 1 directory, 1 pair - so maybe best just go with 
'Make sure to back up the following:'

File: modules/ipi-install-removing-the-provisioner-node.adoc [hunk 5]
```diff
@@ -0,0 +1,13 @@
...
+
+. The `install-config.yaml` file in case the cluster must be redeployed
+. The generated `<kubeconfig>` and `<kubeadmin-passwd>` from the `</path/to/install/dir/auth/>` directory
+. The SSH key pair to SSH to a node
```
Comment (ogradyp) [inline modules/ipi-install-removing-the-provisioner-node.adoc:13]: I would include 'required' here for clarity and so read
The SSH key pair required to SSH to a node.

File: modules/ipi-install-provisioning-the-bare-metal-node.adoc (hunk 3 above)
Comment (ktothill) [reply modules/ipi-install-provisioning-the-bare-metal-node.adoc]: Updated as suggested.

File: modules/ipi-install-removing-the-provisioner-node.adoc (hunk 4 above)
Comment (ktothill) [reply modules/ipi-install-removing-the-provisioner-node.adoc]: Updated as suggested.

File: modules/ipi-install-removing-the-provisioner-node.adoc (hunk 5 above)
Comment (ktothill) [reply modules/ipi-install-removing-the-provisioner-node.adoc]: Updated as suggested.

**Review by kdrake20 — COMMENTED** [review]
Fixing build error.

**Comment by ktothill** [comment]
Closing and recreating.

**Comment by ktothill** [comment]
Reopening

**Comment by ktothill** [comment]
Closing and recreating.

**Comment by ktothill** [comment]
Closing and recreating.

**Pull request closed without merging**